### PR TITLE
standardize / improve naming

### PR DESCRIPTION
**Pull request by jwmueller**
*Issue #, if available:*

https://github.com/awslabs/autogluon/issues/722

### CODE REVIEW

**Comment by szha** [comment]
Job PR-755-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/2/index.html

**Comment by szha** [comment]
Job PR-755-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/3/index.html

**Comment by szha** [comment]
Job PR-755-4 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/4/index.html

**Comment by szha** [comment]
Job PR-755-5 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/5/index.html

**Comment by szha** [comment]
Job PR-755-6 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/6/index.html

**Comment by szha** [comment]
Job PR-755-7 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/7/index.html

**Comment by szha** [comment]
Job PR-755-8 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/8/index.html

**Comment by szha** [comment]
Job PR-755-9 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/9/index.html

**Comment by szha** [comment]
Job PR-755-10 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/10/index.html

**Comment by szha** [comment]
Job PR-755-11 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/11/index.html

**Comment by szha** [comment]
Job PR-755-12 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/12/index.html

**Comment by szha** [comment]
Job PR-755-13 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/13/index.html

**Comment by szha** [comment]
Job PR-755-14 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/14/index.html

**Comment by szha** [comment]
Job PR-755-15 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/15/index.html

**Comment by szha** [comment]
Job PR-755-16 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/16/index.html

**Comment by szha** [comment]
Job PR-755-17 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-755/17/index.html

File: docs/index.rst [hunk 1]
```diff
@@ -1,12 +1,12 @@
-AutoGluon: AutoML Toolkit for Tabular, Text, and Image Data
-===========================================================
+AutoGluon: AutoML for Text, Image, and Tabular Data
+====================================================
 
 `AutoGluon` enables easy-to-use and easy-to-extend AutoML with a focus on automated stack ensembling, deep learning, and real-world applications spanning tabular, text, and image data. Intended for both ML beginners and experts, `AutoGluon` enables you to:
```
Comment (Innixma) [inline docs/index.rst:4]: also reorder text/image/tabular in this sentence

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <img src="https://user-images.githubusercontent.com/16392542/77208906-224aa500-6aba-11ea-96bd-e81806074030.png" width="350">
 </div>
 
-## AutoML Toolkit for Tabular, Text, and Image Data
+## AutoML for Text, Image, and Tabular Data
```
Comment (Innixma) [inline README.md:7]: also reorder text/image/tabular later in this README (first paragraph)

File: docs/tutorials/course/object.md [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
-A user may want to define some customized objects with search spaces, such as network architectures,
-or specialized optimizers. We provide an API to do that.
+When defining custom Python objects such as network architectures,
+or specialized optimizers, it may be hard to decide what values to set for all of their attributes. AutoGluon provides an API that allows you to instead specify  a search space of possible values to consider for such attributes, within which the optimal value will be automatically searched for at runtime. This tutorial demonstrates how easy this is to do, without having to modify your existing code at all!  
 
 ## Example for Constructing a Network
 
-This is an example of doing architecture search as HPO. If you are interested in efficient neural
-architecture search, please refer another tutorial `sec_proxyless`_ .
+This tutorial covers an example of selecting a neural network's architecture as a hyperparameter optimization (HPO) task. If you are interested in efficient neural architecture search (NAS), please refer another tutorial `sec_proxyless`_ .
```
Comment (Innixma) [inline docs/tutorials/course/object.md:9]: 'please refer another tutorial' could be rephrased to be grammatically correct.

**Review by Innixma — APPROVED** [review]
Looks good, added minor comments

**Pull request merged**